src/js/select-text.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function selectText(element) {
+	let range;
+	let selection;
+
+	if (document.body.createTextRange) { //support IE
+		range = document.body.createTextRange();
+		range.moveToElementText(element);
+		range.select();
+	} else if (window.getSelection) { //support everything else
+		selection = window.getSelection();
+		range = document.createRange();
+		range.selectNodeContents(element);
+		selection.removeAllRanges();
+		selection.addRange(range);
+	}
+}
+
+module.exports = (function() {
+	const buttons = document.querySelectorAll('.select-html');
+	if (buttons) {
+		buttons.forEach(button => {
+			button.addEventListener('click', (e) => {
+				e.preventDefault;
+				selectText(button.nextElementSibling);
```

Comment:
Maybe it would be better not to assume the next element is the one we need and instead use a more specific selector?

Reply:
Any suggestions? The next element for these specific buttons is the element that holds the html...